packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Toolbar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content alignment. Align to start, end, or center.
+   */
+  align: PropTypes.oneOf(['start', 'end', 'center']),
+  /**
+   * Items to be displayed in toolbar such as buttons, button groups, and links.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  align: 'start',
```

Comment:
Is the `start` alignment expected more often than not?

Reply:
Yeah. That's going to be case for MPages as well.